### DISCUSSION TITLE
how do I make the character to move in Y axis?

### DISCUSSION THREAD

**Discussion by ckdlskgkdrhd** [naverz/zepeto-studio-global#52] [Q&A]
so thanks alot to @naverz-mkao, I was able to make a working conveyor belt.
and now I'm here with an extended inquiry which might be similar to the belt itself.
I'm trying to make a conveyor belt that would take the character up vertically.
*no I don't want to make an elevator with a mesh that goes up and down, that's boring*
instead, I want to make something like this->
![제목 없음](https://user-images.githubusercontent.com/110361648/184805764-8f071f02-8c79-46d6-9d0b-9a4cf6c5b9cb.png)
by entering this silo, the character is pushed upward until it reaches a certain Y threshold or ceiling.
I assumed the ZepetoCharacterController.Move would work in Y value but it just won't move (probably in-built gravity screwing me)

**Comment by naverz-mkao** [comment]
Hi!

Glad I was able to help you with the previous question!

So to move your character on the Y axis, what you are doing is correct. You just need to make sure that your y-axis amount is greater than the default gravity amount, which is set to 10.

An alternative option is you could set your gravity to 0 (ZepetoPlayers.gravity = 10) while you are in your silo.

```import { CharacterController, Debug, Input, KeyCode, Time, Vector3 } from 'UnityEngine';
import { LocalPlayer, SpawnInfo, ZepetoCharacter, ZepetoPlayers } from 'ZEPETO.Character.Controller';
import { ZepetoScriptBehaviour } from 'ZEPETO.Script'

export default class CharacterLoader extends ZepetoScriptBehaviour {
    private localCharacter: ZepetoCharacter;
    private localCharacterController: CharacterController;

    Start() {        
        ZepetoPlayers.instance.CreatePlayerWithZepetoId("", "[ZEPETO_ID]", new SpawnInfo(), true);
  
        ZepetoPlayers.instance.OnAddedLocalPlayer.AddListener(() => {
            let _player : LocalPlayer = ZepetoPlayers.instance.LocalPlayer;

            this.localCharacter = ZepetoPlayers.instance.LocalPlayer.zepetoPlayer.character;
            this.localCharacterController = this.localCharacter.GetComponent<CharacterController>();

            Debug.LogError(ZepetoPlayers.gravity);  //default gravity is set to 10
        });
    }

    Update() {
        if (Input.GetKey(KeyCode.T)) {

            this.localCharacterController.Move(new Vector3(0,11 * Time.deltaTime,0));
        }
    }

}

**Comment by ckdlskgkdrhd** [comment]
LOL! I can't believe I never bothered to fiddle with the Y speed, thanks! @naverz-mkao
